### PR TITLE
Windows: Take 2

### PR DESCRIPTION
**Pull request by Tobias-Fischer**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by Tobias-Fischer** [comment]
@conda-forge-admin, please rerender

**Comment by Tobias-Fischer** [comment]
So it seems like this builds fine. It is based on #7. It creates and installs a cunit.lib, but not a cunit.dll. I'm not a Windows person and have no idea whether there should be a dll, too. I commented out some stuff in the CMakeLists.txt as it was throwing an error.

Could you have a look and let me know what you think @traversaro @wolfv?

**Comment by wolfv** [comment]
I think we just have to turn this option on `BUILD_SHARED_LIBS`. I can push that change

File: recipe/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,85 @@
+cmake_minimum_required(VERSION 3.0.2)
+project(CUnit LANGUAGES C)
+
+# Export all symbols when building a DLL on MSVC
+if(WIN32 AND MSVC AND BUILD_SHARED_LIBS)
```
Comment (wolfv) [inline recipe/CMakeLists.txt:5]: @Tobias-Fischer here :)

**Comment by wolfv** [comment]
thanks for picking this up, let's get it over the finish line this time :)

**Comment by wolfv** [comment]
dll is built, but installed into `lib`, instead of `bin`. It should be installed into `bin`. I am checking.

**Comment by Tobias-Fischer** [comment]
Do we have to comment the stuff in the CMakeLists back in?

**Comment by wolfv** [comment]
ah, no I think pdb files are for debugging. I just changed `RUNTIME DESTINATION` from `lib` to `bin`. Let's see if that does the trick.

**Comment by traversaro** [comment]
The last problem is that the check for existence checks for `libcunit.lib`, but the default CMake convention for names of `add_library(cunit)` (or also `find_library(cunit)` ) is:
* Unix : `libcunit.<ext>`
* Windows: `cunit.<ext>`

**Comment by traversaro** [comment]
> 
> 
> ah, no I think pdb files are for debugging. I just changed `RUNTIME DESTINATION` from `lib` to `bin`. Let's see if that does the trick.

FYI the `DESTINATION` argument is not required anymore since CMake 3.14 (see https://cmake.org/cmake/help/v3.14/command/install.html#targets). However, the default value uses `CMAKE_INSTALL_LIBDIR` (if `GNUInstallDirs` was previously included) that default to `lib64` on non-Debian distros, so for conda-forge use is important to set it to `lib` explicitly.

**Pull request merged**